src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$main-f-color: #060b35;
+$secondary-f-color: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  color: $main-f-color;
+}
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    max-width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__header {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__text {
+    margin: 0;
+
+    &--color--secondary {
+      color: $secondary-f-color;
+    }
+
+    &--fz--s {
+      font-size: 10px;
+      line-height: 14px;
+    }
+
+    &--fz--m {
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &--fz--l {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__product-code-box {
+    margin-bottom: 16px;
+    align-self: start;
+  }
+
+  &__review-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__link {
+    width: 166px;
+    height: 40px;
+    margin: 0;
+
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+
+    background-color: $blue-accent;
+
+    border-radius: 5px;
+    border: 1px transparent #fff;
+
+    &:hover {
+      color: $blue-accent;
+      border: 1px solid $blue-accent;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover